src/server/local/index.js

```diff
@@ -54,13 +51,6 @@ export default server => {
       const [shouldRedirect, redirectUrl] = removeTrailingSlash(req.url);
       return shouldRedirect ? res.redirect(301, redirectUrl) : next();
     })
-    .use(
-      expressStaticGzip(PUBLIC_DIRECTORY, {
-        enableBrotli: true,
-        orderPreference: ['br'],
-        redirect: false,
-      }),
-    )
```

Comment:
Would this need to stay, just with `enableBrotli: true` removed?

Reply:
This looks like it could be the reason behind the failing Cypress tests (thank goodness we have them - integration tests wouldn't have caught this 😌 ). I think we can also remove the order preference - I am assuming it means br(otli)?

Reply:
could be - weird that everything works fine on my w/c

I'll revert this bit and see what happens with tests

Reply:
You might see this locally if you run the server using `yarn start` rather than `yarn dev` - I am sure that bundling is different because of the need for hotupdate logic in dev mode.